src/include/cephfs/ceph_ll_client.h

```diff
@@ -15,6 +15,10 @@
 #define CEPH_CEPH_LL_CLIENT_H
 #include <stdint.h>
 
+#ifdef _WIN32
+#define AT_SYMLINK_NOFOLLOW 0x100
+#endif
```

Comment:
Ditto here -- might be best to just define this if AT_SYMLINK_NOFOLLOW is not defined.